### DISCUSSION TITLE
Best Model Import Methods for Stride

### DISCUSSION THREAD

**Discussion by najak3d** [stride3d/stride#1294] [Q&A] [answered]
First off, we like to incorporate Blender into our pipeline, to serve as a QA checkpoint, so that we can examine the details of a model, and provide feedback to the artist, in case there are issues.    Then from Blender we currently export to FBX and use an Asset Importer for our proprietary engine to convert to our game engine format.

In Stride, what are the best current methods doing this?
1. From Blender
2. From FBX, or DAE.

And is there usually follow-up manual work to fix-textures?    Our current method with Urho3D has issues with correctly converting FBX to our game engine material system, thus producing some manual work.

**Comment by tebjan** [comment, marked as answer]
See this video tutorial for the basic workflow: https://youtu.be/55WSNO3YHos

Everything is configurable after import. In general fbx and obj work well.
.blend files are directly supported up to blender 2.79, but newer ones will be supported when the assimp bindings are updated.

so at the moment, exporting fbx or obj files from blender should work best.

**Reply by najak3d** [reply]
Can links to a collection of videos like the one you posted here be inserted into the Docs pages? (if not already)

Each link should show a screenshot from the video, to give a visual of what users will find.    When a user asks this type of question, ideally, the answer should instead be a link to a Stride doc page, with links.   Easy-to-do, and great for improving Stride's perception of completeness, especially for newcomers.  Stride deserves an excellent newcomer perception.
